package/libs/openssl/Makefile

```diff
@@ -154,6 +155,10 @@ ifndef CONFIG_OPENSSL_WITH_SRP
   OPENSSL_OPTIONS += no-srp
 endif
 
+ifdef CONFIG_OPENSSL_OPTIMIZE_SPEED
```

Comment:
I'd prefer `ifeq ($(CONFIG_OPENSSL_OPTIMIZE_SPEED),y)` here.

Reply:
I don't mind but that would be inconsistent compared to the rest of the Makefile?

Reply:
i like the ifeq too.